include/uapi/sound/sof/abi.h

```diff
@@ -26,7 +26,7 @@
 
 /* SOF ABI version major, minor and patch numbers */
 #define SOF_ABI_MAJOR 3
-#define SOF_ABI_MINOR 19
+#define SOF_ABI_MINOR 22
```

Comment:
why 22?

Reply:
We already have a pending PR that modifies the ABI. 

https://github.com/thesofproject/linux/pull/3353

Reply:
we should have a separate patch that readjusts ABI level.